addons/calendar/models/calendar_event.py

```diff
@@ -1296,3 +1296,24 @@ def _get_public_fields(self):
             'id', 'active', 'allday',
             'duration', 'user_id', 'interval', 'partner_id',
             'count', 'rrule', 'recurrence_id', 'show_as', 'privacy'}
+
+    def _notify_get_recipients_groups(self, message, model_description, msg_vals=None):
+        attendee = [new_attendee for new_attendee in self.attendee_ids if new_attendee.partner_id.id in msg_vals['partner_ids']]
+        groups = super()._notify_get_recipients_groups(message, model_description, msg_vals=msg_vals)
+        customer_group = next(group for group in groups if group[0] == 'customer')
+        customer_group[2]['active'] = True
+        customer_group[2]['has_button_access'] = True
+        access_opt = customer_group[2].setdefault('button_access', {'title': _('View')})
+        access_opt['url'] = self.get_base_url()+'/calendar/meeting/view?token=%s&id=%s' % (attendee[0].access_token or uuid.uuid4().hex, self.id)
```

Comment:
So the mailing actions will only work for the first attendee of the event? Makes no sense.

Also I don't see why we are creating random uuid here, how would that do anything?